packages/graphql/src/schema/query.ts

```diff
@@ -331,7 +361,8 @@ export const Query = builder.queryType({
           return ParticipantService.getParticipations(args, ctx)
         },
       }),
-      getCourseOverviewData: asAuthenticated.field({
+
+      getCourseOverviewData: asParticipant.field({
```

Comment:
Changed `asAuthenticated` to `asParticipant` as this course query is only used in `frontend-pwa`